tests/api.c

```diff
@@ -33105,6 +33111,78 @@ static int test_wolfSSL_PEM_PrivateKey(void)
     return 0;
 }
 
+static int test_wolfSSL_PEM_file_RSAKey(void)
```

Comment:
Can we merge test_wolfSSL_PEM_read_RSA_PUBKEY and test_wolfSSL_PEM_read_RSAPublicKey with this test?